paddle/optimizer/Tensor_test.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
```

Comment:
`CopyRights` declaration can be added when we merge these modules together. I noticed that go module part all need to add that header section.